content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -282,6 +282,16 @@ on the Kubernetes API server for each static Pod.
 This means that the Pods running on a node are visible on the API server,
 but cannot be controlled from there.
 
+## Container probes
+
+A Probe is a diagnostic performed periodically by the kubelet on a Container. To perform a diagnostic, the kubelet calls a Handler implemented by the container. There are three types of handlers:
+
+- ExecAction
+- TCPSocketAction
+- HTTPGetAction
```

Comment:
How about:
```suggestion
- `ExecAction` (performed with the help of the container runtime)
- `TCPSocketAction` (checked directly by the kubelet)
- `HTTPGetAction` (checked directly by the kubelet)
```
?

Reply:
Looks perfect. I have added the suggested changes.